libmscore/rendermidi.cpp

```diff
@@ -731,6 +735,478 @@ bool Score::isSubdivided(ChordRest* chord, int swingUnit)
             return false;
       }
 
+void renderTremolo(Chord *chord, QList<NoteEventList> & ell){
```

Comment:
Apparently here are the controls xcode allows me.

I’ve changed the tab width to 6.
Any idea what the others mean and how I should set them?




> 
> //---------------------------------------------------------
> //   functionName
> //    optional explanation on multiple line if needed
> //---------------------------------------------------------
> Regarding the indentation, we use 6 spaces to indent. (the code is 4 spaces indented in your PR) The { after a functionName are on a new line. We use if (nospace) { and idem for while, case etc... I started to reindent it so don't bother for this one.
>